logstore/logstore.go

```diff
@@ -244,6 +256,27 @@ func (ls *logstore) getLog(id thread.ID, lid peer.ID) (info thread.LogInfo, err
 	return
 }
 
+// GetOwnLogs returns the logs the host is directly managing under the given thread.
+func (ls *logstore) GetOwnLogs(id thread.ID) ([]thread.LogInfo, error) {
+	logs, err := ls.LogsWithKeys(id)
+	if err != nil {
+		return nil, err
+	}
+	var direct []thread.LogInfo
+	for _, lid := range logs {
+		lg, err := ls.GetLog(id, lid)
+		if err != nil {
+			return nil, err
+		}
+		// We are 'managing' any direct logs or logs for which we have the private key
+		if lg.Direct || lg.PrivKey != nil {
```

Comment:
We return the log if its directly managed by this peer, or if we have the private key. In theory, if private key is available, then it _should_ also be direct.

Reply:
Just based on the comments you have in code explaining "direct", maybe "managed" is a better word? Might not take as much explaining... just a thought.